src/main/webapp/app/exercises/quiz/participate/quiz-participation.component.scss

```diff
@@ -4,15 +4,13 @@
 }
 
 .quiz-content {
-    min-height: calc(100vh - 400px);
-
     .question-index {
```

Comment:
I can't see that the css class `.question-index` is used as well in the `quiz-participation.component.html`. Most probably it can be removed as well.